packages/itwin/tree-widget/src/test/trees/category-tree/CategoriesTreeComponent.test.tsx

```diff
@@ -1,326 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
- * See LICENSE.md in the project root for license terms and full copyright notice.
- *--------------------------------------------------------------------------------------------*/
-
-import { expect } from "chai";
-import { Children } from "react";
-import sinon from "sinon";
-import * as moq from "typemoq";
-import { PropertyRecord } from "@itwin/appui-abstract";
-import { UiFramework } from "@itwin/appui-react";
-import { BeEvent } from "@itwin/core-bentley";
-import { IModelApp, NoRenderApp } from "@itwin/core-frontend";
-import { StandardNodeTypes } from "@itwin/presentation-common";
-import { Presentation } from "@itwin/presentation-frontend";
-import * as treeHeader from "../../../components/tree-header/TreeHeader";
-import * as categoryTree from "../../../components/trees/category-tree/CategoriesTree";
-import { CategoriesTreeComponent } from "../../../components/trees/category-tree/CategoriesTreeComponent";
-import * as categoryVisibilityHandler from "../../../components/trees/category-tree/CategoryVisibilityHandler";
-import { TreeWidget } from "../../../TreeWidget";
-import { act, mockPresentationManager, mockViewport, render, TestUtils, waitFor } from "../../TestUtils";
-
-import type { CategoriesTreeHeaderButtonProps, CategoryInfo } from "../../../components/trees/category-tree/CategoriesTreeButtons";
-import type { Id64String } from "@itwin/core-bentley";
-import type { IModelConnection, Viewport } from "@itwin/core-frontend";
-import type { TreeHeaderProps } from "../../../components/tree-header/TreeHeader";
-import type { IFilteredPresentationTreeDataProvider, PresentationTreeNodeItem } from "@itwin/presentation-components";
-import type { ECInstancesNodeKey } from "@itwin/presentation-common";
-import type { TreeNodeItem } from "@itwin/components-react";
-
-describe("<CategoriesTreeComponent />", () => {
```

Comment:
Should we keep these tests? For the `ModelsTreeComponent` unit tests were kept.

Reply:
added them back